run_checks.sh

```diff
@@ -4,6 +4,6 @@ echo -e "\n##############################"
 echo      "# Running mdl (markdownlint) #"
 echo      "##############################"
 # disable the line-length check
-mdl -r ~MD013,~MD024 -g .
+mdl -r ~MD013,~MD024,~MD007 -g .
```

Comment:
`~MD046` is set in `travis.yml` . Do we want these to match?

Reply:
strangely, without it, on my laptop MD046 was not reported, but I'll keep it consistent.